### DISCUSSION TITLE
Script Error: Object doesn't support property or method 'startsWith'

### DISCUSSION THREAD

**Discussion by Mrinnaal** [cefsharp/CefSharp#4493] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

112.3.0

### Operating System

Windows 10

### Architecture

x64

### .Net Version

.NET 4.8

### Implementation

WPF

### Reproduction Steps

just opening a html document through CEF

### Expected behavior

Should load the links without showing script error popup

### Actual behavior

Shows script error with message as mentioned in the issue title

### Does this problem also occur in the CEF Sample Application

Not Tested

**Comment by amaitland** [comment]
Converted to discussion as this is unlikely a bug in `CefSharp`.

Please test with the `MinimalExample` and the `CefClient` application as detailed in the initial bug report template.

> Should load the links without showing script error popup

To the best of my knowledge `CEF` doesn't implement any sort of `script error popup`. `CefSharp` circinately doesn't implement one.

There's insufficient information or testing completed here.
